<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目一览</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/17844/Desktop/Python大作业/1_主界面.ico</normaloff>C:/Users/17844/Desktop/Python大作业/1_主界面.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>项目进度管理</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1251</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>项目编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>项目类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>项目名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>开始日期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>结束日期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>完成进度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>负责人</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>项目级别</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>备注</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pButtonWatch">
         <property name="text">
          <string>查看当前项目</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pButtonSelect">
           <property name="text">
            <string>搜索项目</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pButtonExport">
           <property name="text">
            <string>导出项目</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pButtonshow">
           <property name="text">
            <string>显示项目及数据</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pButtonExit">
           <property name="text">
            <string>退出系统</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionMenuOpen"/>
    <addaction name="actionMenuExit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionquestion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTlbExit"/>
   <addaction name="actionTlbSelect"/>
   <addaction name="actionTlbWatch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMenuOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionMenuExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionTlbExit">
   <property name="text">
    <string>退出系统</string>
   </property>
  </action>
  <action name="actionTlbSelect">
   <property name="text">
    <string>搜索项目</string>
   </property>
  </action>
  <action name="actionTlbWatch">
   <property name="text">
    <string>查看当前项目</string>
   </property>
  </action>
  <action name="actionquestion">
   <property name="text">
    <string>出现错误？</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
